src/frontend/api/fagsak.ts

```diff
@@ -48,7 +53,7 @@ export const hentFagsaker = (
                 filter,
             },
             method: 'GET',
-            url: '/familie-ba-sak/api/fagsak',
+            url: '/familie-ba-sak/api/fagsaker',
         },
```

Comment:
I don't think this endpoint exists in familie-ba-sak.

Reply:
No, I guess this might be implemented in the future. I will check with Henning H

Reply:
It is not implemented, but we need it pretty soon - so just leave it there 👍